client/message/messages.js

```diff
@@ -68,21 +66,5 @@ Messages.propTypes = {
   messages: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
 }
 
-const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
-
-const getGroupMessages = (state, {params}) => {
-  const groupUid = params.groupUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByGroup(state, groupUid))),
-  }
-}
-
-const getStoryMessages = (state, {params}) => {
-  const storyUid = params.storyUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByStory(state, storyUid))),
-  }
-}
-
 export const GroupMessages = connect(getGroupMessages)(Messages)
```

Comment:
k, got this in place but like with other notes it doesn't seem to be working.  Can you let em know what I'm missing?